### PR TITLE
chore: update faq

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

Related to https://github.com/chaos-mesh/chaos-mesh/pull/2544.

1. Fix zh faq, seems some content is old and not synced from en faq.
2. Add docker client related problem.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 1]
```diff
@@ -2,81 +2,87 @@
...
 ```
 
-You can try using the parameter: `hostNetwork`, as shown below:
+You can use the `hostNetwork` parameter to fix this issue as follows:
 
-```yaml
-# vim helm/chaos-mesh/values.yaml, change hostNetwork from false to true
-hostNetwork: true
-```
+<PickHelmVersion>{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
```
Comment (STRRL) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:21]: This answer with only `helm upgrade` operations might make confusion for users.

How about appending several descriptions about the problem? Like:

This issue is caused by `chaos-controller-manager` failing to connect to `chaos-daemon` (an agent on each Node deployed by DaemonSet), please check the Pod Networking and NetworkPolicy of the kubernetes cluster and firewall rules on Node. And you could try to setup `chaos-daemon` with `hostNetwork`: 

 > The origin answer.

Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:21]: rest lgtm

Comment (g1eny0ung) [reply i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:21]: Agreed. I will update later.

Comment (g1eny0ung) [reply i18n/zh/docusaurus-plugin-content-docs/current/faqs.md]: Done in c195007.

**Review by STRRL — APPROVED** [review]
LGTM

File: docs/faqs.md [hunk 2]
```diff
@@ -13,12 +13,16 @@ No. Instead, you could use [`chaosd`](https://github.com/chaos-mesh/chaosd/) to
 ### Q: I have deployed Chaos Mesh and created PodChaos experiments successfully, but I still failed in creating NetworkChaos/TimeChaos Experiment. The log is shown as below:
 
 ```console
-2020-06-18T01:05:26.207Z	ERROR	controllers.TimeChaos	failed to apply chaos on all pods	{"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxxx: connect: connection refused\""}
+2020-06-18T02:49:15.160Z ERROR controllers.TimeChaos failed to apply chaos on all pods {"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxx: connect: connection refused\""}
 ```
 
-You can use the `hostNetwork` parameter to fix this issue as follows:
+This is caused by `chaos-controller-manager` failing to connect to `chaos-daemon`, you can first check the pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```
Comment (en-jin19) [inline docs/faqs.md:19]:
```suggestion
The reason is that `chaos-controller-manager` failed to connect to `chaos-daemon`. You need to first check the Pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```

File: docs/faqs.md [hunk 3]
```diff
@@ -13,12 +13,16 @@ No. Instead, you could use [`chaosd`](https://github.com/chaos-mesh/chaosd/) to
...
 
-<PickHelmVersion className="language-bash">{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
+If everything is in order, maybe you can use the `hostNetwork` parameter to fix this problem as follows:
+
+<PickHelmVersion>{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
+
+Ref: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```
Comment (en-jin19) [inline docs/faqs.md:25]:
```suggestion
Reference: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```

File: docs/faqs.md [hunk 4]
```diff
@@ -55,9 +59,16 @@ roleRef:
 
 The `USER_ACCOUNT` above should be your Google Cloud user email.
 
+### Q: Daemon throws an error similar to `version 1.41 is too new. Maximum supported API version is 1.39`
+
+This indicates that the maximum API version that the docker daemon can accept is `1.39`, but the client in chaos-daemon uses `1.41` by default. You can choose two options to solve this problem:
+
+1. Upgrade your docker to a newer version.
+2. Helm install/upgrade with `--set chaosDaemon.env.DOCKER_API_VERSION=1.39`.
```
Comment (en-jin19) [inline docs/faqs.md:67]:
```suggestion
### Q: Daemon throws an error similar to `version 1.41 is too new. The maximum supported API version is 1.39`

This indicates that the maximum API version that the Docker daemon can accept is `1.39`, but the client in `chaos-daemon` uses `1.41` by default. You can choose the following options to solve this problem:

1. Upgrade your Docker to a newer version.
2. Helm install/upgrade with `--set chaosDaemon.env.DOCKER_API_VERSION=1.39`.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 5]
```diff
@@ -2,81 +2,91 @@
...
+### Q: I have deployed Chaos Mesh and created PodChaos experiments successfully, but I still failed in creating NetworkChaos/TimeChaos Experiment. The log is shown as below:
 
-```bash
-kind delete cluster
+```console
+2020-06-18T02:49:15.160Z ERROR controllers.TimeChaos failed to apply chaos on all pods {"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxx: connect: connection refused\""}
 ```
 
-then deploy again.
+This is caused by `chaos-controller-manager` failing to connect to `chaos-daemon`, you can first check the pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:19]:
```suggestion
The reason is that `chaos-controller-manager` failed to connect to `chaos-daemon`. You need to first check the Pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 6]
```diff
@@ -2,81 +2,91 @@
...
 
-## Debug
+If everything is in order, maybe you can use the `hostNetwork` parameter to fix this problem as follows:
 
-### Q: Experiment not working after chaos is applied
+<PickHelmVersion>{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
 
-You can debug as described below:
+Ref: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:25]:
```suggestion
Reference: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 7]
```diff
@@ -2,81 +2,91 @@
...
+  apiGroup: rbac.authorization.k8s.io
+```
 
-  ```bash
-  kubectl logs -n chaos-testing chaos-controller-manager-xxxxx (replace this with the name of the controller-manager) | grep "ERROR"
-  ```
+The `USER_ACCOUNT` above should be your Google Cloud user email.
 
-  For error message `no pod is selected`, run the following command to show the labels and check if the selector is desired.
+### Q: Daemon throws an error similar to `version 1.41 is too new. Maximum supported API version is 1.39`
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:62]:
```suggestion
### Q: Daemon throws an error similar to `version 1.41 is too new. The maximum supported API version is 1.39`
```

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 8]
```diff
@@ -2,81 +2,91 @@
...
 
-  ```bash
-  kubectl get pods -n yourNamespace --show-labels
-  ```
+This indicates that the maximum API version that the docker daemon can accept is `1.39`, but the client in chaos-daemon uses `1.41` by default. You can choose two options to solve this problem:
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:64]:
```suggestion
This indicates that the maximum API version that the Docker daemon can accept is `1.39`, but the client in `chaos-daemon` uses `1.41` by default. You can choose the following options to solve this problem:
```

File: i18n/zh/docusaurus-plugin-content-docs/current/faqs.md [hunk 9]
```diff
@@ -2,81 +2,91 @@
...
 
-If the above steps cannot solve the problem or you encounter other related errors in controller's log, [file an issue](https://github.com/chaos-mesh/chaos-mesh/issues) or message us in the #project-chaos-mesh channel in the [CNCF Slack](https://join.slack.com/t/cloud-native/shared_invite/zt-fyy3b8up-qHeDNVqbz1j8HDY6g1cY4w) workspace.
+1. Upgrade your docker to a newer version.
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/faqs.md:66]:
```suggestion
1. Upgrade your Docker to a newer version.
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/faqs.md [hunk 10]
```diff
@@ -2,81 +2,91 @@
...
+### Q: I have deployed Chaos Mesh and created PodChaos experiments successfully, but I still failed in creating NetworkChaos/TimeChaos Experiment. The log is shown as below:
 
-```bash
-kind delete cluster
+```console
+2020-06-18T02:49:15.160Z ERROR controllers.TimeChaos failed to apply chaos on all pods {"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxx: connect: connection refused\""}
 ```
 
-then deploy again.
+This is caused by `chaos-controller-manager` failing to connect to `chaos-daemon`, you can first check the pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/faqs.md:19]:
```suggestion
The reason is that `chaos-controller-manager` failed to connect to `chaos-daemon`. You need to first check the Pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/faqs.md [hunk 11]
```diff
@@ -2,81 +2,91 @@
...
 
-## Debug
+If everything is in order, maybe you can use the `hostNetwork` parameter to fix this problem as follows:
 
-### Q: Experiment not working after chaos is applied
+<PickHelmVersion>{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
 
-You can debug as described below:
+Ref: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/faqs.md:25]:
```suggestion
Reference: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```

File: versioned_docs/version-2.0.5/faqs.md [hunk 12]
```diff
@@ -13,12 +13,16 @@ No. Instead, you could use [`chaosd`](https://github.com/chaos-mesh/chaosd/) to
 ### Q: I have deployed Chaos Mesh and created PodChaos experiments successfully, but I still failed in creating NetworkChaos/TimeChaos Experiment. The log is shown as below:
 
 ```console
-2020-06-18T01:05:26.207Z	ERROR	controllers.TimeChaos	failed to apply chaos on all pods	{"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxxx: connect: connection refused\""}
+2020-06-18T02:49:15.160Z ERROR controllers.TimeChaos failed to apply chaos on all pods {"reconciler": "timechaos", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp xx.xx.xx.xx:xxxx: connect: connection refused\""}
 ```
 
-You can use the `hostNetwork` parameter to fix this issue as follows:
+This is caused by `chaos-controller-manager` failing to connect to `chaos-daemon`, you can first check the pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```
Comment (en-jin19) [inline versioned_docs/version-2.0.5/faqs.md:19]:
```suggestion
The reason is that `chaos-controller-manager` failed to connect to `chaos-daemon`. You need to first check the Pod network and its [policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
```

File: versioned_docs/version-2.0.5/faqs.md [hunk 13]
```diff
@@ -13,12 +13,16 @@ No. Instead, you could use [`chaosd`](https://github.com/chaos-mesh/chaosd/) to
...
 
-<PickHelmVersion className="language-bash">{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
+If everything is in order, maybe you can use the `hostNetwork` parameter to fix this problem as follows:
+
+<PickHelmVersion>{`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n chaos-testing --version latest --set chaosDaemon.hostNetwork=true`}</PickHelmVersion>
+
+Ref: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```
Comment (en-jin19) [inline versioned_docs/version-2.0.5/faqs.md:25]:
```suggestion
Reference: <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#hostport-services-do-not-work>
```

File: versioned_docs/version-2.0.5/faqs.md [hunk 14]
```diff
@@ -55,9 +59,16 @@ roleRef:
 
 The `USER_ACCOUNT` above should be your Google Cloud user email.
 
+### Q: Daemon throws an error similar to `version 1.41 is too new. Maximum supported API version is 1.39`
+
+This indicates that the maximum API version that the docker daemon can accept is `1.39`, but the client in chaos-daemon uses `1.41` by default. You can choose two options to solve this problem:
+
+1. Upgrade your docker to a newer version.
+2. Helm install/upgrade with `--set chaosDaemon.env.DOCKER_API_VERSION=1.39`.
```
Comment (en-jin19) [inline versioned_docs/version-2.0.5/faqs.md:67]:
```suggestion
### Q: Q: Daemon throws an error similar to `version 1.41 is too new. The maximum supported API version is 1.39`

This indicates that the maximum API version that the Docker daemon can accept is `1.39`, but the client in `chaos-daemon` uses `1.41` by default. You can choose the following options to solve this problem:

1. Upgrade your Docker to a newer version.
2. Helm install/upgrade with `--set chaosDaemon.env.DOCKER_API_VERSION=1.39`.
```

**Comment by g1eny0ung** [comment]
@en-jin19 Thank u so much for all suggestions 😃, I've already resolved all comments, PTAL again.

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**